evaluators/embedding/MinkowskiDistanceEvaluator/__init__.py

```diff
@@ -0,0 +1,34 @@
+import numpy as np
+
+from jina.executors.evaluators.embedding import BaseEmbeddingEvaluator, expand_vector
+
+
+class MinkowskiDistanceEvaluator(BaseEmbeddingEvaluator):
+    """A :class:`MinkowskiDistanceEvaluator` evaluates the distance between actual and desired 
+    embeddings computing the Minkowski distance ( p>0 ) between them, which can be considered 
+    as a generalization of both the Euclidean distance and the Manhattan distance.
+
+        D(x, y) = ( sum_i |x_i - y_i|^p )^(1/p)
+    """
+
+    @property
+    def metric(self):
+        return 'MinkowskiDistanceEvaluator'
+
+    def evaluate(self, actual: 'np.array', desired: 'np.array', *args, **kwargs) -> float:
+        """"
+        :param actual: the embedding of the document (resulting from an Encoder)
+        :param desired: the expected embedding of the document
+        :return the evaluation metric value for the request document
+        """
+        actual = expand_vector(actual)
+        desired = expand_vector(desired)
+        order = kwargs["order"]
+        return _minkowski_Distance(actual, desired, order)
+
+
+def _minkowski_Distance(actual, desired, order):
+    if order <= 0:
+        raise ValueError("Order must be positive.")
+    minkowskiDistExp = np.sum(np.power(np.abs(actual - desired), order))
```

Comment:
Underscore case to be used here

Reply:
<img width="219" alt="Screen Shot 2020-11-09 at 10 41 57 PM" src="https://user-images.githubusercontent.com/61920576/98599772-d2d11780-22dc-11eb-97ed-7310b1659579.png">

sorry it is my first time to pr. I have tried to set the username and user email. And I am not sure if it works or not.

Reply:
Now you need to change your commit messages

Reply:
it turns out that my Git email is another one.